src/readDicomElementExplicit.js

```diff
@@ -1,6 +1,6 @@
 import findEndOfEncapsulatedElement from './findEndOfEncapsulatedPixelData.js';
 import findAndSetUNElementLength from './findAndSetUNElementLength.js';
-import findItemDelimitationItemAndSetElementLength from './findItemDelimitationItem.js';
```

Comment:
You still need this import because this function is called below for parsing an item (i.e. an undefined length element that's not a sequence and not UN).